typescript/infra/scripts/funding/fund-keys-from-deployer.ts

```diff
@@ -539,19 +542,19 @@ class ContextFunder {
   }
 
   private async bridgeIfL2(chain: ChainName) {
-    if (L2Chains.includes(chain)) {
+    if (L2Chains.includes(chain as L2Chain)) {
```

Comment:
would prefer not using the "as" cast here

Reply:
alternate would be to do `(L2Chains as string[])` instead

Reply:
also why not cast here, as we're casting a few lines later

Reply:
rejigged stuff but it means we're doing a nasty `as readonly string[]` when switching depending on the L2 chain bridge we need

Reply:
ok, this works then